compiler/src/iree/compiler/Dialect/Flow/Transforms/RegionOpUtils.cpp

```diff
@@ -367,7 +376,7 @@ Flow::clonePrecedingOpIntoDispatchRegion(RewriterBase &rewriter,
     });
   }
 
-  return newTargetOp;
+  return success();
```

Comment:
Still better to return the op... The caller might need to do "something" with the new op.

Reply:
Fixed.